scripts/nayduck.py

```diff
@@ -50,6 +50,10 @@ def _parse_args():
                        '-i',
                        action='store_true',
                        help='Read test set from standard input.')
+    group.add_argument('--dry-run',
```

Comment:
this should be added to `parser`, not `group`, i.e. `parses.add_argument(…)`. Options within `group` are mutually exclusive.